app/services/stats_summary.rb

```diff
@@ -10,6 +10,7 @@ def as_slack_message
       :pencil: #{pluralize(applications_begun(today), 'application')} begun | #{applications_begun(this_day_last_year)} last cycle
       :#{mailbox_emoji(applications_submitted(today))}: #{pluralize(applications_submitted(today), 'application')} submitted | #{applications_submitted(this_day_last_year)} last cycle
       :#{rand(2) == 1 ? 'wo' : nil}man-tipping-hand: #{pluralize(offers_made(today), 'offer')} made | #{offers_made(this_day_last_year)} last cycle
+      :#{rand(2) == 1 ? 'wo' : nil}man-tipping-hand: #{pluralize(offers_accepted(today), 'offer')} accepted | #{offers_accepted(this_day_last_year)} last cycle
```

Comment:
obviously it doesn't matter but `%w(man woman).sample` would be a lot more legible

Reply:
🧹